src/sketcher/helper.tsx

```diff
@@ -0,0 +1,84 @@
+import { Button } from '@jupyterlab/ui-components';
+import * as React from 'react';
+
+import { IPosition } from './types';
+
+export function drawPoint(
+  ctx: CanvasRenderingContext2D,
+  point: IPosition,
+  fillStyle = 'blue',
+  size = 6
+): void {
+  const oldFill = ctx.fillStyle;
+  ctx.beginPath();
+  ctx.fillStyle = fillStyle;
+  ctx.fillRect(point.x - size / 2, point.y - size / 2, size, size);
+  ctx.closePath();
+  ctx.fillStyle = oldFill;
```

Comment:
There is no need for a path here

```suggestion
  ctx.fillStyle = fillStyle;
  ctx.fillRect(point.x - size / 2, point.y - size / 2, size, size);
  ctx.fillStyle = oldFill;
```